hw/ip/hmac/data/hmac.hjson

```diff
@@ -304,16 +304,20 @@
 
             Message is byte granularity.
             lower 3bits [2:0] are ignored.''',
-      swaccess: "ro",
-      hwaccess: "hwo",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      hwext: "true",
+      hwqe: "true",
       fields: [
         { bits: "31:0", name: "v", desc: "Message Length [31:0]" }
       ]
     }
     { name: "MSG_LENGTH_UPPER",
       desc: "Received Message Length calculated by the HMAC in bits [63:32]",
```

Comment:
I suggest updating these comments to specify that software can also write those and more importantly **when** software can write them.

Reply:
Thx, done